src/test/resources/mocks/iac-daily-list/iacDailyList.json

```diff
@@ -68,7 +68,6 @@
                             "partyRole": "CLAIMANT_PETITIONER_REPRESENTATIVE",
                             "individualDetails": {
                               "individualForenames": "Individual Forenames",
-                              "individualMiddleName": "Individual Middlename",
```

Comment:
There's a few more IAC test files that these will be removed from. 

There's a couple more in this repo (in the functionalTest/resources folder), and also some in test folders in the frontend repo, and test folders in the channel management repo